### ISSUE TITLE
[🔴URGENT] NYON STATUE' SUPPLY BURN SUGGESTION

### ISSUE THREAD

**Issue by Killu8726** [sunflower-land/sunflower-land#653]
Analyzing other Decorative Nfts in the game, we can see that it maintains almost the same value linked to its supply within the game, because it does not have a functionality, it ends up being not such a requested NFT, causing them to have low values ​​in the Open Sea, which ends up in a way completely discouraging the creation of these NFTs because you spend a higher value in resource (open sea quote) to create an item that is worth less. I know that one of the goals of those NFTs is burning tokens and resources, but the fact is: putting a high supply for NFTs with a high value in resources like wood, stone, iron, gold and SFL can generate a devaluation over time, since once there isn't a way to craft it, what will determinate its price will be only the supply. I think it would be interesting to burn Supply from the Nyon statue making it have Supply maximum 300 to 500 units, to encourage players to make it even with the high cost it has, and become a more scarce and valued item. We have to consider that scarcity of some resources is also an important thing in every game and remember this might affect everyone.

![image](https://user-images.githubusercontent.com/103577122/164792580-e7b925c3-69bf-48bb-a576-abd8c70b2390.png)

**Comment by Kubeki** [comment]
Decoration NFTs are losing interest, and farmers are beginning to understand that decorative NFTs won't be of any use to them, that's normal.

**Comment by F1rezd** [comment]
> Os NFTs decorativos estão perdendo o interesse e os agricultores estão começando a entender que os NFTs decorativos não serão úteis para eles, isso é normal.

If you are true, the cat wouldn't be one of the most expensive nfts, you said nonsense, the question is the Supply, the Decorative NFTs can be worth much more than the ones that have a function

**Comment by Killu8726** [comment]
> > Os NFTs decorativos estão perdendo o interesse e os agricultores estão começando a entender que os NFTs decorativos não serão úteis para eles, isso é normal.
> 
> If you are true, the cat wouldn't be one of the most expensive nfts, you said nonsense, the question is the Supply, the Decorative NFTs can be worth much more than the ones that have a function

Yes, that's what Im talking about. There is only 64 Farm Cats in market, what proves my point: the lower the supply is, the higher is the price. There must be a balance between material cost and supply, otherwise they will not be worth crafting and people will lose interest overtime.

**Comment by SFL-Aeon** [comment]
The only reason the cat is expensive is because it’s one of the original v1 NFTs. Many people couldn’t mint it. Also many people like the cat. Art is subjective and if people don’t want to buy the NFT they don’t have to. If you bought the statue then hey enjoy it but the supply won’t be changed because you want to market manipulate. 

The devs already said if you want to make something make it but the game isn’t going to follow the market. If you bought the statue while it was high then you made a choice but that is what it is a choice. 

Once the market stables out and the NFTs mint out then cost may go up it may go down. 

many people say there is not enough NFTs and making a small supply so the early minters, the hoarders, the P2P traders get them all is not the goal for the devs.

**Comment by Killu8726** [comment]
> The only reason the cat is expensive is because it’s one of the original v1 NFTs. Many people couldn’t mint it. Also many people like the cat. Art is subjective and if people don’t want to buy the NFT they don’t have to. If you bought the statue then hey enjoy it but the supply won’t be changed because you want to market manipulate.
> 
> The devs already said if you want to make something make it but the game isn’t going to follow the market. If you bought the statue while it was high then you made a choice but that is what it is a choice.
> 
> Once the market stables out and the NFTs mint out then cost may go up it may go down.
> 
> many people say there is not enough NFTs and making a small supply so the early minters, the hoarders, the P2P traders get them all is not the goal for the devs.

I don't even have the statue for "manipulation" the market, you're not understanding the point of this burning suggestion. Showing the price is just a clear example of its value, it doesn't really matter if it is about the price or not, I'm saying for the value itself. There should be a better balance for material cost and supply.

**Comment by SFL-Aeon** [comment]
@Killu8726 I think the point should’ve included if the # to mint was going to be higher the cost to mint should’ve been lower. 

I agree with this statement the $50SFL could’ve been 20, 10 or even 0. The costs could’ve been looked at better yet these costs are cheaper than what was proposed.

when I meant you I meant whomever would want to reduce a set supply after it’s released. That is market manipulation. Asking the devs to do it is why I referred as you.

**Comment by SteveChickenCX** [comment]
I don't see why it's "URGENT"

I also don't see any reason to reduce the supply of a cosmetic NFT, In a few halving crafting this NFT would become unfeasible ,naturally limiting its supply even if there are plenty left .

I agree with Aeon, Changing the mint of an NFT after releasing it is wrong.

**Comment by Kubeki** [comment]
> > Os NFTs decorativos estão perdendo o interesse e os agricultores estão começando a entender que os NFTs decorativos não serão úteis para eles, isso é normal.
> 
> If you are true, the cat wouldn't be one of the most expensive nfts, you said nonsense, the question is the Supply, the Decorative NFTs can be worth much more than the ones that have a function

As mentioned before the cat is an NFT of V1 so it has obviously increased in value but on top of that the DEVs had promised that the cat would have a use in the game, the same for dog.

**Comment by adamhannigan** [comment]
Thanks for the contribution and feedback! ❤️ 

 We are moving towards using an ideas portal to help capture and vote on community ideas. Github will only be used for bug/technical reports. We are only a small team and it is becoming difficult for us to respond individually to every case so we have began using community voting software.   

 If you could please upload your idea to the following community portal it would be greatly appreciated 🤗   

 https://sunflower-land.ideas.aha.io/

 You can vote on similar ideas and help bring ideas to our attention. Thanks again for being apart of this community!
